helm/prometheus-rules/templates/platform/cabbage/alerting-rules/network.rules.yml

```diff
@@ -1,124 +0,0 @@
-apiVersion: monitoring.coreos.com/v1
-kind: PrometheusRule
-metadata:
-  creationTimestamp: null
-  labels:
-    {{- include "labels.common" . | nindent 4 }}
-  name: network.rules
-  namespace: {{ .Values.namespace  }}
-spec:
-  groups:
-  - name: network
-    rules:
-    - alert: DNSErrorRateTooHigh
```

Comment:
Created a dns rule file for cabbage and sent the provider independant ones to turtles for now